docs/conf.py

```
@@ -36,6 +37,7 @@ def setup(app):
     app.add_css_file(
         "https://cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.2.2/gh-fork-ribbon.min.css"
     )
+    app.add_config_value("is_development_build", False, "env")
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)